.github/workflows/main.yml

```diff
@@ -35,6 +35,20 @@ jobs:
               run: |
                   scripts/local-cluster-setup.sh
                   scripts/dev-setup.sh
+            - name: Start local cluster
+              run: |
+                  docker-compose -f scripts/local_cluster/docker-compose.yml up -d
```

Comment:
Quick note here: `docker-compose.yml` is not a script. This is a configuration file, so I'd be careful not to mix too many scripts and config files in the `scripts` folder

Reply:
Are you suggesting moving it somewhere else?  It's technically not a script by the strict definition, but pretty strongly tied to the `local-cluster-*` scripts, so it's not clear to me that we would gain anything by moving it.  I've changed these lines to not reference the file directly, so everything should be abstracted behind the `local-cluster-*` scripts.  But let me know if you prefer a different approach.  Thanks.

Reply:
> pretty strongly tied to the local-cluster-* scripts, so it's not clear to me that we would gain anything by moving it. I've changed these lines to not reference the file directly, so everything should be abstracted behind the local-cluster-* scripts.

Makes sense, thanks. If the config file is only used by one script, that makes sense to keep it like that. My eye was simply caught when reading the CI config and seeing that some commands used config files in the `scripts` repo, which didn't seem appropriate.